steamapi.cpp

```diff
@@ -1,5 +1,4 @@
 #include <string.h>
```

Comment:
I don't see `#include <string.h>` is necessary in this file. Neither `malloc' nor `str*` CRT functions are used there.

Reply:
I'm not sure. It is not really what I wanted to be looking into for this PR.

Reply:
I will try to look into it when I am done with these other PRs.